packages/orchestrator-ui-components/src/messages/nl-NL.json

```diff
@@ -183,7 +183,14 @@
             "created": "Aangemaakt",
             "endDate": "Eind datum",
             "note": "Notitie",
-            "noRelatedSubscriptions": "Geen gerelateerde subscriptions gevonden"
+            "noRelatedSubscriptions": "Geen gerelateerde subscriptions gevonden",
+            "hideTerminatedRelatedSubscriptions": "Hide terminated related subscriptions",
```

Comment:
I don't like EN text in the NL language file.

Reply:
Aangepast